src/components/Navbar/Navbar.vue

```diff
@@ -0,0 +1,1259 @@
+	height: 24px;
+	padding: 0;
+	margin: auto;
+	transition: color 150ms;
+	cursor: pointer;
+}
+
+header .locale-dropdown-toggle:hover,
+header .locale-dropdown-toggle.active {
+	color: var(--text-shade-3);
+}
+
+header .user-widget-wrapper {
+	height: auto;
+}
+
+header .user-widget-wrapper a.login-link {
+	color: var(--text-shade-1);
+	text-decoration: none;
+	display: block;
+	height: 32px;
+	transition: color 150ms;
+}
+
+header .user-widget-wrapper a.login-link:hover {
+	color: var(--text-shade-3);
+}
+
+header .user-widget-wrapper.logged-in {
+	position: relative;
+	width: 32px;
+	height: 32px;
+}
+
+header .user-widget-wrapper.logged-in .user-widget-toggle {
+	width: 32px;
+	height: 32px;
+	background: var(--text-shade-0);
+	border-radius: 50%;
+	overflow: hidden;
+	cursor: pointer;
+}
+
+header .user-widget-wrapper .user-widget-toggle img,
+header .user-widget .user-avatar img {
+	width: 100%;
+	height: 100%;
+}
+
+header .user-widget {
+	max-height: 0;
+	overflow: hidden;
+
+	box-sizing: border-box;
+	transition: max-height 300ms, padding 200ms, opacity 150ms;
+
+	position: absolute;
+	right: 0;
+	top: 48px;
+	padding: 0;
+	background: var(--bg-shade-2);
+	border-radius: 8px;
+	text-align: center;
+	opacity: 0;
+
+	box-shadow: 0 0 10px -2px var(--bg-shade-0);
+}
+
+header .user-widget.active {
+	max-height: 100vh;
+	padding: 36px;
+	opacity: 1;
+}
+
+header .user-widget .user-avatar {
+	width: 128px;
+	height: 128px;
+	margin: auto;
+	background: var(--text-shade-0);
+	border-radius: 50%;
+	overflow: hidden;
+}
+
+header .user-widget .user-info {
+	margin-top: 12px;
+}
+
+header .user-widget .user-info .mii-name {
+	color: var(--text-shade-3);
+}
+
+header .user-widget .buttons {
+	margin-top: 12px;
+}
+
+header .user-widget .button {
+	margin-top: 12px;
+	width: 100%;
+	padding: 8px 60px;
+	cursor: pointer;
+}
+
+.locale-dropdown-toggle {
+	appearance: none;
+	background: none;
+	border: none;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 32px;
+	height: 32px;
+	padding: 0;
+	margin: 0;
+	transition: color 150ms;
+	cursor: pointer;
+}
+
+.locale-dropdown-toggle:hover,
+.locale-dropdown-toggle.active {
+	color: var(--text-shade-3);
+}
+
+.options-container {
+	position: absolute;
+	top: 100%;
+	right: 0;
+
+	display: flex;
+	flex-flow: column;
+	gap: 2px;
+	width: min(90vw, 240px);
+	max-height: min(90vh, 20rem);
+	overflow-y: scroll;
+
+	margin-top: 24px;
+
+	border-radius: 5px;
+	background-color: var(--bg-shade-3);
+	order: 1;
+}
+
+.option {
+	all: unset;
+	font: inherit;
+	color: var(--text-shade-2);
+	padding: 12px 15px;
+	cursor: pointer;
+	border-radius: 5px;
+}
+
+.option:hover {
+	background: var(--bg-shade-4);
+}
+
+.locale-dropdown {
+	position: relative;
+	width: min-content;
+	height: fit-content
+}
+
+header .user-widget .button.logout {
+	background: var(--bg-shade-3);
+	color: var(--text-shade-3);
+}
+
+@media screen and (max-width: 900px) {
+	header {
+		position: relative;
```

Comment:
I must admit I haven't tried it before. But I don't think media queries work in a scoped style block?
Simple solution is to just make another `<style>` (not scoped) and put all the media queries in there.

Reply:
i am a fool